canary/producer-cpp/scripts/generate-iot-credentials.sh

```diff
@@ -48,8 +55,10 @@ cat > iam-permission-document.json <<EOF
         {
             "Effect": "Allow",
             "Action": [
-                "kinesisvideo:ListStreams",
-                "kinesisvideo:CreateStream"
+                "iot:Receive",
+                "iot:Subscribe",
+                "iot:Connect",
+                "iot:Publish"
             ],
```

Comment:
I see that `iot:Allow` is present in the `iot-policy-document.json` but not here. Is it intentional?